src/modules/global_vars/__init__.py

```diff
@@ -3,11 +3,11 @@
 CONFIG_RULES = {
```

Comment:
`CONFIG_RULES` corrections should be fixed up to b5190492921025f4806e42db88fc8f0b8324aa6e.